ENH: Add information on how to set up the projector (#155)
TheAxonLab/hcph-sops: docs/data-collection/pre-session.md

- [ ] Go to room {{ secrets.rooms.projector | default("███") }}, where the projector is installed.
- [ ] Switch the projector **ON** by hitting the power button, located on its right side.

> !!! important "The projector's position is already finalized. Only make changes if it is seriously off. If that is the case, adjust it as follows."

Reviewer: ```suggestion
!!! important "The projector's position is already finalized. Only make changes if it is seriously off and as a last resort. If that is the case, adjust it as follows."
```

> !!! important "The projector's position is already finalized. Only make changes if it is seriously off and as a last resort. If that is the case, adjust it as follows."

Reviewer 2: ```suggestion

    !!! important "The projector's position SHALL NOT be modified unless it is evidently off"

        Only in the case that the projector beam is not correctly targeting the projection screen inside the Scanning Room, which is possible if someone accidentally altered the configuration, you may adjust its position as follows:
```

> 
>     - [ ] Adjust the projector tilt to center the projection if it does not properly aim the panel inside the scanner's bore.
>       E.g., change the height of the paper pile that supports it (see images, FENS papers).
> 
>       | ![projector](../assets/images/projector.png) | ![paper-projector](../assets/images/paper-projector.png) | ![adjust-projector](../assets/images/adjust-projector.png) |
>       |:--:|:--:|:--:|
>       | *The hole is the part through which you should check the quality of the projection* {: colspan=3} | | |

Reviewer 2: ```suggestion

          - [ ] Adjust the projector tilt to center the projection if it does not properly aim the panel inside the scanner's bore.
      E.g., change the height of the paper pile that supports it (see images, FENS papers).

            | ![projector](../assets/images/projector.png) | ![paper-projector](../assets/images/paper-projector.png) | ![adjust-projector](../assets/images/adjust-projector.png) |
            |:--:|:--:|:--:|
            | *The hole is the part through which you should check the quality of the projection* {: colspan=3} | | |
```

- [ ] Go to room {{ secrets.rooms.projector | default("███") }}, where the projector is installed.
- [ ] Switch the projector **ON** by hitting the power button, located on its right side.

    

>     !!! important 
>         "The projector's position SHALL NOT be modified unless it is evidently off"
>     
>     Only in the case that the projector beam is not correctly targeting the projection screen inside the Scanning Room, which is possible if someone accidentally altered the configuration, you may adjust its position as follows:
>     
>         - [ ] Adjust the projector tilt to center the projection if it does not properly aim the panel inside the scanner's bore.
>            E.g., change the height of the paper pile that supports it (see images, FENS papers).
>         

Reviewer 2: This is not equivalent to my suggestion. Perhaps this will render well?
```suggestion
    !!! important "The projector's position SHALL NOT be modified unless it is evidently off"

        Only in the case that the projector beam is not correctly targeting the projection screen inside the Scanning Room, which is possible if someone accidentally altered the configuration, you may adjust its position as follows:

        - [ ] Adjust the projector tilt to center the projection if it does not correctly aim the panel inside the scanner's bore.
           E.g., change the height of the paper pile that supports it (see images, FENS papers).

```

            | ![projector](../assets/images/projector.png) | ![paper-projector](../assets/images/paper-projector.png) | ![adjust-projector](../assets/images/adjust-projector.png) |
            |:--:|:--:|:--:|
            | *The hole is the part through which you should check the quality of the projection* {: colspan=3} | | |

- [ ] Verify that the projector projects your laptop screen by looking through the window of the console room.
- [ ] Before you exit room {{ secrets.rooms.projector | default("███") }}, go to the next step.

- [ ] Verify that the projector projects your laptop screen by looking through the window of the console room.
- [ ] Before you exit room {{ secrets.rooms.projector | default("███") }}, go to the next step.

> 

Reviewer 2: Duplicated? why?

Reviewer 2: fixed
